app/src/main/res/layout/activity_connection_test.xml

```diff
@@ -0,0 +1,120 @@
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Working"
+        android:textSize="80dp"
+        android:textColor="@color/green"
+        android:layout_marginLeft="8dp"
+        android:layout_marginRight="8dp"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        android:layout_marginTop="8dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        android:layout_marginBottom="8dp"
+        app:layout_constraintHorizontal_bias="0.622"
+        app:layout_constraintVertical_bias="0.739"
+        app:layout_constraintLeft_toRightOf="@+id/ping_label_textView"/>
+
+    <TextView
+        android:id="@+id/http_label_textView"
+        android:layout_width="173dp"
+        android:layout_height="95dp"
+        android:text="http:"
+        android:textSize="80dp"
+        android:layout_marginLeft="8dp"
+        app:layout_constraintLeft_toLeftOf="parent"
+        android:layout_marginRight="8dp"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        android:layout_marginTop="8dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        android:layout_marginBottom="8dp"
+        app:layout_constraintHorizontal_bias="0.1"
+        app:layout_constraintVertical_bias="0.478"/>
+
+    <TextView
+        android:id="@+id/ping_label_textView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="8dp"
+        android:layout_marginLeft="8dp"
+        android:layout_marginRight="8dp"
+        android:layout_marginTop="8dp"
+        android:text="ping:"
+        android:textAlignment="viewStart"
+        android:textSize="80dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintHorizontal_bias="0.1"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.739"/>
+
+
+    <TextView
+        android:id="@+id/http_result_textView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/working"
+        android:textSize="80dp"
+        android:textColor="@color/green"
+        android:layout_marginLeft="8dp"
+        android:layout_marginRight="8dp"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        android:layout_marginTop="8dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        android:layout_marginBottom="8dp"
+        app:layout_constraintHorizontal_bias="0.618"
+        app:layout_constraintVertical_bias="0.478"
+        app:layout_constraintLeft_toRightOf="@+id/http_label_textView"/>
+
+</android.support.constraint.ConstraintLayout>
```

Comment:
I can't actually CR a ConstraintLayout... As we said, I think it is unnecessary here... Anyway pay attention to the order of elements in the xml- It's arranged weirdly.